### PR TITLE
5951 create a command to trigger the import of a single message

### LINKED ISSUE #5951: Create a command to trigger the import of a single message
- Add `messageId` in log when there is an error importing a message
- In `packages/twenty-server/src/modules/messaging/message-import-manager` add a command to trigger the import of a single message. This command takes as an argument `messageId`, `connectedAccountId` and `workspaceId`.

### PR DESCRIPTION
**Pull request by bosiraphael**
Closes #5951

### CODE REVIEW

File: packages/twenty-server/src/modules/messaging/common/services/messaging-fetch-by-batch.service.ts [hunk 1]
```diff
@@ -32,19 +38,25 @@ export class MessagingFetchByBatchesService {
...
 
+      messageIdsByBatch.push(
+        messageIds.slice(batchOffset, batchOffset + batchLimit),
+      );
+
       batchOffset += batchLimit;
     }
 
-    return batchResponses;
+    return { messageIdsByBatch, batchResponses };
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/messaging/common/services/messaging-fetch-by-batch.service.ts:48]: LGTM but I would even return a singleMap: { messageId, response }

File: packages/twenty-server/src/modules/messaging/message-import-manager/commands/messaging-single-message-import.command.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    flags: '-m, --message-id [message_id]',
+    description: 'Message ID',
+    required: true,
+  })
+  parseMessageId(value: string): string {
+    return value;
+  }
+
+  @Option({
+    flags: '-n, --message-channel-id [message_channel_id]',
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/messaging/message-import-manager/commands/messaging-single-message-import.command.ts:53]: why n? :p

File: packages/twenty-server/src/modules/messaging/message-import-manager/jobs/messaging-add-single-message-to-cache-for-import.job.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { CacheStorageService } from 'src/engine/integrations/cache-storage/cache-storage.service';
+import { InjectCacheStorage } from 'src/engine/integrations/cache-storage/decorators/cache-storage.decorator';
+import { CacheStorageNamespace } from 'src/engine/integrations/cache-storage/types/cache-storage-namespace.enum';
+import { Process } from 'src/engine/integrations/message-queue/decorators/process.decorator';
+import { Processor } from 'src/engine/integrations/message-queue/decorators/processor.decorator';
+import { MessageQueue } from 'src/engine/integrations/message-queue/message-queue.constants';
+import { MessagingChannelSyncStatusService } from 'src/modules/messaging/common/services/messaging-channel-sync-status.service';
+
+export type MessagingAddSingleMessageToCacheForImportJobData = {
+  messageId: string;
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/messaging/message-import-manager/jobs/messaging-add-single-message-to-cache-for-import.job.ts:10]: should this be externalMessageId?

File: packages/twenty-server/src/modules/messaging/message-import-manager/commands/messaging-single-message-import.command.ts (hunk 2 above)
Comment (bosiraphael) [reply packages/twenty-server/src/modules/messaging/message-import-manager/commands/messaging-single-message-import.command.ts:53]: For some reason -c wasn't working

File: packages/twenty-server/src/modules/messaging/message-import-manager/jobs/messaging-add-single-message-to-cache-for-import.job.ts (hunk 3 above)
Comment (bosiraphael) [reply packages/twenty-server/src/modules/messaging/message-import-manager/jobs/messaging-add-single-message-to-cache-for-import.job.ts:10]: Yes

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**